android/app/src/main/java/app/organicmaps/widget/placepage/PlacePageController.java

```diff
@@ -230,13 +230,20 @@ private void stopCustomPeekHeightAnimation()
   private void onHiddenInternal()
   {
     if (ChoosePositionMode.get() == ChoosePositionMode.None)
-      Framework.nativeDeactivatePopup();
+      Framework.nativeDeactivatePopup(true);
     Framework.nativeDeactivateMapSelectionCircle(false);
     PlacePageUtils.updateMapViewport(mCoordinator, mDistanceToTop, mViewportMinHeight);
     resetPlacePageHeightBounds();
     removePlacePageFragments();
   }
 
+  private void onTrackRecordingSelected()
+  {
+    if (ChoosePositionMode.get() == ChoosePositionMode.None)
+      Framework.nativeDeactivatePopup(false);
```

Comment:
This is the only place in the code where you call this function with `false` argument. Please, find a better solution. Do you really need this false? Can it be done differently?